FirebaseRemoteConfig/Tests/FakeConsole/FakeConsoleTests.swift

```diff
@@ -59,6 +60,43 @@ class FakeConsoleTests: APITestBase {
     waitForExpectations()
   }
 
+  // Test New API.
+  // Contrast with testUnchangedActivateWillFlag in APITests.swift.
+  func testChangedActivateWillNotFlag() {
```

Comment:
Should this also test the `false, nil` case where activate is called but no changes happened?

Reply:
Yes, but that test doesn't need the FakeConsole and was added to APITests.swift